<launch>
	<node pkg="robotic_arm" type="i2c_server.py" name="server_i2c" output="screen">
	</node>
	<node pkg="robotic_arm" type="interface_node.py" name="interface" output="screen">
	</node>
</launch>

